bundlegen/schema/appMetadataSchema.json

```diff
@@ -0,0 +1,48 @@
+{
+    "$schema": "http://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "properties": {
+        "id": {"type": "string"},
+        "type": {"type": "string"},
+        "version": {"type": "string"},
+        "description": {"type": "string"},
+        "priority": {"type": "string"},
+        "graphics": {"type": "boolean"},
+        "network": {
+            "type": "object",
+            "properties": {
+                "type": {"type": "string"}
+            },
+            "required": ["type"]
+        },
+        "storage": {"type": "object"},
+        "resources": {
+            "type": "object",
+            "properties": {
+                "ram": {"type": "string"}
+            },
+            "required": ["ram"]
+        },
+        "features": {
+            "type": "array",
+            "items": {}
+        },
+        "mounts": {
+            "type": "array",
+            "items": {}
+        }
+    },
+    "required": [
```

Comment:
We shouldn't require all these fields - it changes behaviour vs the current documentation: https://github.com/rdkcentral/BundleGen/blob/master/docs/AppMetadata.md